docs/model_developers/resource_sets.rst

```diff
@@ -98,6 +98,10 @@ Two arguments can be passed to the ``inmanta export`` command in order to export
 - ``--partial`` To specify that the model being compiled only contains the resources that need to be updated in relation to the previous version of the model.
 - ``--delete-resource-set <resource-set-name>`` This option, which may be used more than once, instructs the model to remove the resource set with the specified name. Only in conjunction with the preceding choice may this option be utilized. Note that utilizing a ``std::ResourceSet`` that includes no resources allows resource sets to be implicitly deleted during a partial compilation.
 
+.. note::
```

Comment:
I don't see why we need to the note box here. It used to attract the attention of the user to something special. This bit is not something special is it?